src/test/java/bio/terra/workspace/service/job/JobServiceTest.java

```diff
@@ -55,7 +55,6 @@ public void setup() {
     }
   }
 
-  @Test
```

Comment:
Why remove this? It looks like a test to me

Reply:
hah, didn't intend for that to slip through. I was just manually ignoring it during development because it's one of the slower tests. good catch